pkg/process/checks/process.go

```diff
@@ -483,6 +490,11 @@ func fmtProcesses(
 			ProcessContext:         serviceExtractor.GetServiceContext(fp.Pid),
 		}
 
+		if tags, ok := pidToGPUTags[fp.Pid]; ok {
+			log.Debugf("Detected GPU, and process is in activePids, adding GPU tags to pid: %d", fp.Pid)
+			proc.Tags = tags
```

Comment:
Currently these are the only tags we're adding, but let's not make that assumption here in case we add tags to the process in the future before this call happens.

```suggestion
			proc.Tags = append(proc.Tags, tags)
```